fe/fe-core/src/main/java/com/starrocks/transaction/PublishVersionDaemon.java

```diff
@@ -563,6 +595,45 @@ public boolean publishPartitionBatch(Database db, long tableId, long partitionId
         return true;
     }
 
+    private void submitDeleteTxnLogJob(TransactionStateBatch txnStateBatch, Map<Long, List<Long>> dirtyPartitions) {
+        getDeleteTxnLogExecutor().submit(() -> {
```

Comment:
`submit` may throw [RejectedExecutionException](https://docs.oracle.com/javase/8/docs/api/java/util/concurrent/RejectedExecutionException.html) if the task cannot be scheduled for execution.